lib/start-command.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+var debug = require('./debug')('start-command');
+var fs = require('fs');
+var path = require('path');
+
+module.exports = exports = resolveArgs;
+
+// for tests
+exports.fromPath = fromPath;
+exports.resolveFile = resolveFile;
+exports.resolveDir = resolveDir;
+exports.fromStart = fromStart;
+
+function resolveArgs(cwd, argv) {
+  var script = argv[2] || '.';
+  return fromPath(cwd, script);
+}
+
+function fromPath(cwd, script) {
+  var app = {
+    cwd: cwd,
+    path: script,
+    error: null,
+  };
+
+  try {
```

Comment:
the whole block could use some comments, after stareing at it, I think what its doing is:

- if path refers to a directory or to a file, done
- if not, its possible it refers to something that require would resolve to a file, such as `server/server` (no `.js`), so let require.resolve try to figure that out
- at end, path is a file or directory